src/commands/build-util.ts

```diff
@@ -66,8 +67,8 @@ export async function wrapCssModuleResponse({
   hasHmr?: boolean;
   config: SnowpackConfig;
 }) {
-  let core = new Core();
-  const {injectableSource, exportTokens} = await core.load(code, url, () => {
+  _cssModuleLoader = _cssModuleLoader || new (require('css-modules-loader-core'))();
```

Comment:
We need to use `require()` here, since TypeScript's `import` support forces us to grab the `default` export off of the module, but that doesn't actually exist. But, we still get type info from the `CSSModuleLoader` type above.